src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -2031,28 +2031,28 @@ void Player_AnimReplaceNormalPlayLoopAdjusted(PlayState* play, Player* this, Lin
 }
 
 void Player_ProcessControlStick(PlayState* play, Player* this) {
-    s8 phi_v1;
-    s8 phi_v0;
+    s8 spinAngle;
+    s8 direction;
 
     this->prevControlStickMagnitude = sControlStickMagnitude;
     this->prevControlStickAngle = sControlStickAngle;
 
     Lib_GetControlStickData(&sControlStickMagnitude, &sControlStickAngle, sControlInput);
 
-    D_808535DC = Camera_GetInputDirYaw(GET_ACTIVE_CAM(play)) + sControlStickAngle;
+    sControlStickAngleRelToCamera = Camera_GetInputDirYaw(GET_ACTIVE_CAM(play)) + sControlStickAngle;
```

Comment:
`sControlStickAngleRelToCamera` is not relative to the camera, it's the contrary, it takes into account the camera

`sControlStickAngle` is the one that is "relative to the camera": forward (control stick up) is the forward that can be seen on screen

`sControlStickAngleRelToCamera` is the corresponding angle it "translates" to in the world

Reply:
I dont follow. `sControlStickAngle` is just the raw control stick angle calculated from the values given by the `Input` data. I dont think thats relative to the camera in any way.
It takes that absolute angle calculated from the controller and adds the result from `Camera_GetInputDirYaw` to it here.

I guess my understanding gets fuzzy from here, what is "inpu dir yaw" from camera?

Reply:
I spoke with @Thar0 about what "camera input direction" actually means. 
If he finds the time he can weigh in on some more specific details. 

But basically, "Input Dir Yaw" is just the camera's yaw in world space... *usually*. 
Under normal circumstances, it holds the same value as the real camera yaw. 
There are some circumstances where this yaw value gets modified, so it is not the same as the actual camera yaw value. I have not done investigation on all of these cases. But one of them is for the "crit wiggle" feature of the camera. Some other more specialized camera modes change the value a bit to make it differ from the true camera yaw.

I think whoever named this variable "input direction" was kind of naming this in reverse, only accounting for how player uses it. There are other uses not related to "input" necessarily.

Anyway, the name tharo and I agreed on for this specific player variable was `sControlStickWorldYaw`. It is the raw stick angle, plus the angle of the camera in world space. This results in the direction the stick is being pointed in, in world space.

Reply:
"sControlStickAngleRelToCamera is the corresponding angle it "translates" to in the world"
re-reading your message I see you already came to this conclusion as well 😅 

I still think the name of "input dir" is not great and should change, but camera docs are hard